exercises/graphing_inequalities_2.html

```diff
@@ -206,7 +206,7 @@
                         $("input[name='isSolution']:checked").attr("id")]
                     </div>
                     <div class="validator-function">
-                        if (_.isEqual(guess, [[-5, 5], [5, 5], false, true])) {
+                        if (_.isEqual(guess.slice(0, 4), [[-5, 5], [5, 5], false, true]) || guess[4] === null) {
```

Comment:
In any case the guess has 5 elements, so no. A simpler fix is to add `null` as the fifth element to the array, but this is still not how I'd see it.

Logically, this exercise consists of two parts, a graph and a radio selection. It would be nice to reflect this in its structure. I imagine it could look something like this:

```html
<div class="solution" data-type="multiple">
    <div class="sol" data-type="custom">...</div>
    <div class="sol" data-type="radio">...</div>
    <ul class="choices" data-category="true">...</ul>
</div>
```

This is what drove the changes on the other branch (including making the problem unanswered when no part or just one part is answered). At least the two changes related to radio-in-multiple [third commit](https://github.com/Khan/khan-exercises/commit/e7f623cdda7a46aeb248ed103b5a501e92b1f2af) are needed to make it work.